File: Chapter1/OneOne.java
public class OneOne {
    public static void main(String[] args){
        System.out.println(check("hello"));
        System.out.println(check("helo"));
    }
    static boolean check(String s){
        boolean[] unique = new boolean[128];
        char[] characterArray = s.toCharArray();
        for (char c : characterArray){
            if (unique[Character.getNumericValue(c)]){
                return false;
            }
            else {
                unique[Character.getNumericValue(c)] = true;
            }
        }
        return true;
    }
}

File: Chapter1/one-one.py
# check if string has all unqiue characters

def check(s):
    string_values = list(s)
    unique = [False] * 128
    for string in string_values:
        if (unique[ord(string)]):
            return False
        else:
            unique[ord(string)]= True
    return True            

print(check("helo"))